Library/Formula/l-smash.rb

```diff
@@ -1,7 +1,7 @@
 class LSmash < Formula
   desc "Tool for working with MP4 files"
   homepage "https://l-smash.github.io/l-smash/"
-  url "https://github.com/l-smash/l-smash.git", :tag => "v1.13.2", :shallow => false
+  url "https://github.com/l-smash/l-smash.git", :tag => "v2.9.1", :shallow => false
```

Comment:
You'll need to add a `revision` (The upstream commit the tag is based off) here as well now.

```ruby
url "https://github.com/l-smash/l-smash.git", :shallow => false,
     :tag => "v2.9.1",
     :revision => "xyz"
```